transports/sqs/delayed-delivery_enable_sqs_[7.3,).partial.md

```diff
@@ -0,0 +1,4 @@
+In NServiceBus.AmazonSQS version 7.3 and higher, the unrestricted delayed delivery feature is enabled by default. It can be disabled by creating the transport instance using the corresponding contructor accepting a boolean `disableDelayedDelivery` and setting it to `false`.
```

Comment:
```suggestion
The unrestricted delayed delivery feature requires both senders and receivers to enable it. In NServiceBus.AmazonSQS version 7.3 and higher, this feature is enabled by default. It can be disabled by creating the transport instance using the corresponding constructor, accepting a boolean `disableDelayedDelivery`, and setting it to `false`.
```